test/LegalPropositionSpec.hs

```diff
@@ -59,29 +59,29 @@ spec = do
           isJust $ runIdentity $ evalLangM (checkLegalPropositionConfig legalPropConfig :: LangM Maybe)
     describe "illegalDisplay" $ do
         it "at least creates actual formula symbols" $
-            within timeout $ forAll validBoundsSynTree $ \SynTreeConfig {..} ->
+            within timeout $ forAll validBoundsSynTree $ \synTreeConfig@SynTreeConfig {..} ->
                 forAll
                   (genSynTree synTreeConfig) $ \synTree ->
                       forAll (deleteSpaces <$> illegalDisplay synTree) $
                       all (\c -> c `elem` "()∧∨¬<=>" || isLetter c)
         it "the string after illegalDisplay cannot be parsed" $
-            within timeout $ forAll validBoundsSynTree $ \SynTreeConfig {..} ->
+            within timeout $ forAll validBoundsSynTree $ \synTreeConfig@SynTreeConfig {..} ->
```

Comment:
Is the `@SynTreeConfig {..}` part actually still needed? It seems that no access to the fields actually happens below. (Might also be relevant at other places.)

Reply:
No, I removed it.